contributing/index.md

```diff
@@ -11,7 +11,8 @@ Data Commons has benefited greatly from our collaborations with different govern
 
 - [Add data](#add-data): Contribute data to the Data Commons Graph.
 - [Create tools](#creating-a-new-tool): Build new tools or applications that bring the data in Data Commons to new categories of users.
-- [Create new curriculum](#sharing-analysis): Using Data Commons in data science and machine learning courses.
+- [Share analyses](#sharing-analysis): Use Data Commons to build models and share interesting insights.
+- [Create new curriculum](#create-new-curriculum): Use Data Commons in data science and machine learning courses.
```

Comment:
I'd move this up under "add data" and update the ordering of the sections below to match.

Reply:
Done (see new commit)! Incidentally, everything is now in alphabetical order.